### PR TITLE
chore(data-explorer): added spinner on dataload

### PR DESCRIPTION
**Pull request by jelmerveen**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
-  User documentation updated
- [x] Conventional commits (squash if needed)
-  No warnings during install
-  Updated javascript typing

### CODE REVIEW

**Review by MaxPostema — APPROVED** [review]

File: packages/data-explorer/src/views/MainView.vue [hunk 1]
```diff
@@ -87,23 +81,9 @@ export default Vue.extend({
...
-    ...mapState('header', [
-      'breadcrumbs'
-    ]),
-    ...mapGetters([
-      'isUserAuthenticated',
-      'compressedRouteFilter'
-    ]),
+    ...mapState(['filters', 'showSelected', 'dataDisplayLayout', 'tableData', 'tableName', 'tableMeta', 'tableSettings', 'searchText', 'loading']),
+    ...mapState('header', ['breadcrumbs']),
+    ...mapGetters(['isUserAuthenticated', 'compressedRouteFilter']),
```
Comment (connoratrug) [inline packages/data-explorer/src/views/MainView.vue:86]: can we keep the long format , is easer to scan

File: packages/data-explorer/src/views/MainView.vue [hunk 2]
```diff
@@ -113,27 +93,13 @@ export default Vue.extend({
...
-      'deleteRow',
-      'fetchViewData',
-      'fetchTableMeta'
-    ]),
-    ...mapActions('header', [
-      'fetchPackageTables'
-    ]),
+    ...mapMutations(['setHideFilters', 'setTableName', 'setToasts', 'setSearchText', 'setFilterSelection', 'setDataDisplayLayout', 'setRouteQuery']),
+    ...mapActions(['deleteRow', 'fetchViewData', 'fetchTableMeta']),
+    ...mapActions('header', ['fetchPackageTables']),
```
Comment (connoratrug) [inline packages/data-explorer/src/views/MainView.vue:102]: can we keep the long format , is easer to scan

File: packages/data-explorer/src/views/MainView.vue [hunk 3]
```diff
@@ -6,43 +6,37 @@
...
       <div class="mg-filter mr-2">
         <filters-view></filters-view>
       </div>
       <div class="d-flex flex-column mr-2 h-100 overflow-control w-100">
-        <active-filters
-          @input="saveFilterState"
-          :value="activeFilterSelections"
-          :filters="filterDefinitions"
-        ></active-filters>
+        <active-filters @input="saveFilterState" :value="activeFilterSelections" :filters="filterDefinitions"></active-filters>
```
Comment (connoratrug) [inline packages/data-explorer/src/views/MainView.vue:30]: can we keep the long format , is easer to scan

File: packages/data-explorer/src/views/MainView.vue (hunk 1 above)
Comment (MaxPostema) [reply packages/data-explorer/src/views/MainView.vue:86]: - Ran linter -- fix
- A quick look tough the files tels me that most files have 1 liners like this. if there is a wish to make mayor coding changes lets put it in a separate PR ( for now i think its more important to have a consistent styling over all files )

**Review by MaxPostema — APPROVED** [review]

**Comment by connoratrug** [comment]
@MaxPostema i do not agree;

- most files do not have everything on a single line
- consistency is not a end goal, the end goal is maintainable code, formating the code makes is readable (  maintainable)

**Pull request merged**